chain/consensus/filcns/upgrades.go

```diff
@@ -1431,6 +1431,155 @@ func upgradeActorsV8Common(
 	return newRoot, nil
 }
 
+func UpgradeActorsCode(ctx context.Context, sm *stmgr.StateManager, newActorsManifestCid cid.Cid, root cid.Cid, av actors.Version, oldStateTreeVersion types.StateTreeVersion, newStateTreeVersion types.StateTreeVersion) (cid.Cid, error) { // nolint
```

Comment:
It's currently unused. I probably don't need to include the top-level wrapper here for now anyway, since we will need to write a wrapper that puts the new manifest and junk in the store and such

Reply:
I'd leave it, I just don't think anything will get mad at you for having an unused method.